Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -864,9 +724,6 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 		u32 pteg_addr = ((hash & PowerPC::ppcState.pagetable_hashmask) << 6) | PowerPC::ppcState.pagetable_base;
 
-		if ((pteg_addr >> 28) == 1)
-			base_mem = Memory::m_pEXRAM;
-
 		for (int i = 0; i < 8; i++)
 		{
 			u32 pte = bswap(*(u32*)&base_mem[pteg_addr]);
```

Comment:
It might be faster to build a local UPTE1 (from the appropriate VSID/API values, V=1 and H=hash_func) and byteswap it outside of the loop, then compare to the value in memory here. Then you don't have the bitmasking and compares below to make sure it's a match.